docs/flow/concepts/_index.md

```diff
@@ -7,17 +7,6 @@ weight: 100
 
 # Concepts
 
-Grafana Agent Flow has three primary concepts:
+This explain explains primary concepts of Grafana Agent Flow.
```

Comment:
s/explain//. Also, what is the <section> below? I couldn't figure it out with a glance.

Reply:
It's a Hugo shortcode that we implemented for our docs site that generates a list of links.

If you cd to `docs/` and run `make docs` you can see what happens when you navigate to http://localhost:3002/docs/agent/latest